dotnet-8.0.404.yaml

```diff
@@ -0,0 +1,99 @@
+package:
+  name: dotnet-8.0.404
+  version: 8.0.404
+  epoch: 1
+  description: ".NET SDK"
+  copyright:
+    - license: MIT
+  resources:
+    cpu: 2
+    memory: 32Gi
+  dependencies:
+    runtime:
+      - dotnet-8-sdk-default
```

Comment:
@pnasrat @imjasonh @dakaneye note this SDK depends on the default (longterm) SDK and the default runtime. This is not standalone version stream, but a supplement / build-time-only-additional-SDK.